packs/spell-effects/spell-effect-sages-curse-failure.json

```
@@ -0,0 +1,87 @@
+{
+    "_id": "DAAtzP2QYmCzSNXk",
+    "img": "systems/pf2e/icons/spells/savants-curse.webp",
+    "name": "Spell Effect: Sage's Curse (Failure)",
+    "system": {
+        "description": {
+            "value": "<p>The target must roll twice and use the worse result whenever attempting an Arcana, Lore, Occultism, or Society check.</p>"
+        },
+        "duration": {
+            "expiry": null,
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Abomination Vaults Adventure Path"
+        },
+        "rules": [
+            {
+                "keep": "lower",
+                "key": "RollTwice",
+                "predicate": [
+                    {
+                        "or": [
+                            "check:statistic:arcana",
+                            "check:statistic:occultism",
+                            "check:statistic:society"
+                        ]
+                    }
+                ],
+                "removeAfterRoll": false,
+                "selector": [
+                    "skill-check"
+                ]
+            },
+            {
+                "keep": "lower",
+                "key": "RollTwice",
+                "removeAfterRoll": false,
+                "selector": "lore-skill-check"
+            },
+            {
+                "choices": [
+                    {
+                        "label": "Failure",
+                        "value": "failure"
+                    },
+                    {
+                        "label": "Critical Failure",
+                        "value": "criticalFailure"
+                    }
```

```suggestion
                    {
                        "label": "PF2E.Check.Result.Degree.Check.failure",
                        "value": "failure"
                    },
                    {
                        "label": "PF2E.Check.Result.Degree.Check.criticalFailure",
                        "value": "critical-failure"
                    }
```
Need to localize and the values should be kebab case